hazelcast/src/main/java/com/hazelcast/internal/config/ScheduledExecutorConfigReadOnly.java

```diff
@@ -59,4 +61,9 @@ public ScheduledExecutorConfig setMergePolicyConfig(MergePolicyConfig mergePolic
     public ScheduledExecutorConfig setStatisticsEnabled(boolean statisticsEnabled) {
         throw new UnsupportedOperationException("This config is read-only scheduled executor: " + getName());
     }
+
+    @Override
+    public ScheduledExecutorConfig setNamespace(@Nullable String namespace) {
+        throw new UnsupportedOperationException("This config is read-only scheduled executor: " + getName());
```

Comment:
nit: These could be unified across configs - sometimes there is `throwReadOnly`, sometimes it is like this.

Reply:
Agreed, but I'll leave that out of the UCD PR for now - I did think the same when I was looking over the PR changes.